.github/workflows/bundler-audit.yml

```diff
@@ -1,8 +1,8 @@
 name: Bundler Audit
 on:
+  merge_group:
   push:
-    branches-ignore:
-      - 'dependabot/**'
+    branches: [main]
```

Comment:
You may also want to include the Release branches or a regex for them as well

Reply:
The actual releases I think just get tags ... do you mean the various `stable-*` branches? I guess if someone pushed directly to one of those w/out doing so via PR we would want that to run.

Reply:
I think its these https://github.com/mastodon/mastodon/branches/all?query=releases that are created, then opened for backporting

Reply:
Yes, this needs to be enabled for the `stable-*` branches. I am not sure what those `releases` branches are, but they are not used.

Reply:
Updated to include `stable-*` in the branches filter.